modules/nf-core/freyja/variants/meta.yml

```diff
@@ -36,10 +36,6 @@ output:
       type: file
       description: File containing identified variants in a gff-like format
       pattern: "*.variants.tsv"
-  - depths:
-      type: file
-      description: File containing depth of the variants
-      pattern: "*.depth.tsv"
```

Comment:
why delete?

Reply:
Ah, I see, it was removed because nf-core lint generated a warning. This is an an output except it is currently not picked by nf-core lint. I'll add it back again and investigate why the lint generated a warning.

Reply:
This is fixed now and I have raised an issue regarding the warning: https://github.com/nf-core/tools/issues/3097